app/managers.py

```diff
@@ -0,0 +1,47 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")

```

Comment:
It's better to use here `.db3` file extension

Reply:
Can I use .sqlite3 instead .db3?